backend/routers/social_media_marketing_router.py

```
@@ -74,6 +74,8 @@ def get_avatar_engine():
 except ImportError:
     TIKTOK_SERVICE_AVAILABLE = False
 
+import asyncpg
```

Ruff (0.12.2)

Redefinition of unused `asyncpg` from line 17

Remove definition: `asyncpg`

(F811)